pywebdriverd

```diff
@@ -9,7 +9,7 @@ def main():
     if config.getboolean('application', 'print_status_start'):
         if 'escpos' in drivers:
             drivers['escpos'].push_task('printstatus')
-    app.run(host=host, port=port, debug=debug)
+    app.run(host=host, port=port, debug=debug, use_reloader=(not debug))
```

Comment:
I think it's not ok, as it would activate the reloader in production mode. I'll push a commit to make it configurable.